amps/ags/rm-community/rm-community-repo/config/alfresco/module/org_alfresco_module_rm/rm-service-context.xml

```diff
@@ -695,6 +696,14 @@
           init-method="init" depends-on="org_alfresco_module_rm_resourceBundles">
         <property name="policyComponent" ref="policyComponent"/>
         <property name="nodeService" ref="nodeService"/>
+        <!-- list of disposition actions to automatically execute when eligible -->
+        <property name="retentionActions">
+            <list>
+                <value>cutoff</value>
```

Comment:
Shouldn't the other retention actions be listed here?

Reply:
is cut-off needed here? isn't this covered through the security interceptor https://github.com/Alfresco/alfresco-community-repo/blob/master/amps/ags/rm-community/rm-community-repo/config/alfresco/module/org_alfresco_module_rm/rm-action-context.xml#L197? Did you manage to get the message for the cut off action `unable to perform action cut off because the node is frozen or has frozen children`?